src/plugins/initiatives/initiativesMap.js

```diff
@@ -1,7 +1,10 @@
 // @flow
 
+import {type NodeAddressT, NodeAddress} from "../../core/graph";
 import {type Compatible, fromCompat, toCompat} from "../../util/compat";
-import {type URL} from "./initiative";
+import type {Initiative, URL} from "./initiative";
+
+const trackerPrefix = NodeAddress.fromParts(["sourcecred", "initiativesMap"]);
```

Comment:
by convention, the first two entries should be `[organizationPublishingPlugin, pluginName]`, and subsequent pieces are plugin-defined. Since this is our initiatives plugin, it should start with `["sourcecred", "initiatives"]`